doc/manual/install-judgehost.rst

```diff
@@ -128,6 +144,10 @@ Edit grub config to add cgroup memory and swap accounting to the boot
 options. Edit ``/etc/default/grub`` and change the default
 commandline to
 ``GRUB_CMDLINE_LINUX_DEFAULT="quiet cgroup_enable=memory swapaccount=1"``
+Optionally the timings can be more stable by denying the OS to schedule
+other tasks to same CPU the judgedaemon is using:
```

Comment:
Better wording:
"Optionally the timings can be made more stable by not letting the OS schedule
any other tasks on the same CPU core the judgedaemon is using:"